core/dbt/events/types_pb2.py

```diff
@@ -16,3464 +15,10 @@
-_FIRSTRUNRESULTERROR = DESCRIPTOR.message_types_by_name['FirstRunResultError']
-_FIRSTRUNRESULTERRORMSG = DESCRIPTOR.message_types_by_name['FirstRunResultErrorMsg']
-_AFTERFIRSTRUNRESULTERROR = DESCRIPTOR.message_types_by_name['AfterFirstRunResultError']
-_AFTERFIRSTRUNRESULTERRORMSG = DESCRIPTOR.message_types_by_name['AfterFirstRunResultErrorMsg']
-_ENDOFRUNSUMMARY = DESCRIPTOR.message_types_by_name['EndOfRunSummary']
-_ENDOFRUNSUMMARYMSG = DESCRIPTOR.message_types_by_name['EndOfRunSummaryMsg']
-_LOGSKIPBECAUSEERROR = DESCRIPTOR.message_types_by_name['LogSkipBecauseError']
-_LOGSKIPBECAUSEERRORMSG = DESCRIPTOR.message_types_by_name['LogSkipBecauseErrorMsg']
-_ENSUREGITINSTALLED = DESCRIPTOR.message_types_by_name['EnsureGitInstalled']
-_ENSUREGITINSTALLEDMSG = DESCRIPTOR.message_types_by_name['EnsureGitInstalledMsg']
-_DEPSCREATINGLOCALSYMLINK = DESCRIPTOR.message_types_by_name['DepsCreatingLocalSymlink']
-_DEPSCREATINGLOCALSYMLINKMSG = DESCRIPTOR.message_types_by_name['DepsCreatingLocalSymlinkMsg']
-_DEPSSYMLINKNOTAVAILABLE = DESCRIPTOR.message_types_by_name['DepsSymlinkNotAvailable']
-_DEPSSYMLINKNOTAVAILABLEMSG = DESCRIPTOR.message_types_by_name['DepsSymlinkNotAvailableMsg']
-_DISABLETRACKING = DESCRIPTOR.message_types_by_name['DisableTracking']
-_DISABLETRACKINGMSG = DESCRIPTOR.message_types_by_name['DisableTrackingMsg']
-_SENDINGEVENT = DESCRIPTOR.message_types_by_name['SendingEvent']
-_SENDINGEVENTMSG = DESCRIPTOR.message_types_by_name['SendingEventMsg']
-_SENDEVENTFAILURE = DESCRIPTOR.message_types_by_name['SendEventFailure']
-_SENDEVENTFAILUREMSG = DESCRIPTOR.message_types_by_name['SendEventFailureMsg']
-_FLUSHEVENTS = DESCRIPTOR.message_types_by_name['FlushEvents']
-_FLUSHEVENTSMSG = DESCRIPTOR.message_types_by_name['FlushEventsMsg']
-_FLUSHEVENTSFAILURE = DESCRIPTOR.message_types_by_name['FlushEventsFailure']
-_FLUSHEVENTSFAILUREMSG = DESCRIPTOR.message_types_by_name['FlushEventsFailureMsg']
-_TRACKINGINITIALIZEFAILURE = DESCRIPTOR.message_types_by_name['TrackingInitializeFailure']
-_TRACKINGINITIALIZEFAILUREMSG = DESCRIPTOR.message_types_by_name['TrackingInitializeFailureMsg']
-_RUNRESULTWARNINGMESSAGE = DESCRIPTOR.message_types_by_name['RunResultWarningMessage']
-_RUNRESULTWARNINGMESSAGEMSG = DESCRIPTOR.message_types_by_name['RunResultWarningMessageMsg']
-_DEBUGCMDOUT = DESCRIPTOR.message_types_by_name['DebugCmdOut']
-_DEBUGCMDOUTMSG = DESCRIPTOR.message_types_by_name['DebugCmdOutMsg']
-_DEBUGCMDRESULT = DESCRIPTOR.message_types_by_name['DebugCmdResult']
-_DEBUGCMDRESULTMSG = DESCRIPTOR.message_types_by_name['DebugCmdResultMsg']
-_LISTCMDOUT = DESCRIPTOR.message_types_by_name['ListCmdOut']
-_LISTCMDOUTMSG = DESCRIPTOR.message_types_by_name['ListCmdOutMsg']
-_NOTE = DESCRIPTOR.message_types_by_name['Note']
-_NOTEMSG = DESCRIPTOR.message_types_by_name['NoteMsg']
-EventInfo = _reflection.GeneratedProtocolMessageType('EventInfo', (_message.Message,), {
```

Comment:
It's weird that the next 3k lines were removed and not regenerated in any form 🤔

Reply:
I looked into this a bit. Although my changeset removes all these lines, they were not present in older versions of the file. It looks like they may have been introduced unintentionally by [this changeset](https://github.com/dbt-labs/dbt-core/commit/4c44c29ee4cec511f4751cc9fec668949ff9726b).